spec/bundle/insecure_sources/Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'http://rubygems.org'
 
-gem 'rails'
+gem 'rails', '~> 5.2'
```

Comment:
This doesn’t seem related. Can this be removed?

Reply:
removed